### DISCUSSION TITLE
I need help to run ./nb-dt-import.py behind a proxy server

### DISCUSSION THREAD

**Discussion by orlipr84** [netbox-community/devicetype-library#2270] [General]
hello friends,

I need to install the device types, but since I am behind a proxy server, the ./nb-dt-import.py script fails and I cannot import the device types that they have in the library. Could you help me? Thank you.

**Comment by harryajc** [comment]
Hi @orlipr84 
Please raise this on the [Device-Type-Library-Import](https://github.com/netbox-community/Device-Type-Library-Import) as the script is part of that repo not this one.
